packages/web/integrations/axelar/types.ts

```diff
@@ -22,53 +22,82 @@ export interface AxelarBridgeConfig {
- */
+/** Intermediate source chain identifiers */
 export type SourceChain =
-  | "aurora"
+  | "Aurora Testnet"
   | "Avalanche"
-  | "binance"
+  | "Avalanche Fuji Testnet"
+  | "Binance Smart Chain"
+  | "BSC Testnet"
   | "Ethereum"
-  | "ethereum-2"
+  | "Goerli Testnet"
   | "Fantom"
+  | "Fantom Testnet"
   | "Moonbeam"
-  | "Polygon";
+  | "Moonbase Alpha"
+  | "Polygon"
+  | "Mumbai";
 
-/** Maps eth client chainIDs => axelar chain ids.
- *
- *  ethClientChainIDs must be specified in ../ethereuem/types.ts::ChainNames{} to map the name to a chainID, which is in turn used to add the network to EVM-compatible wallets, like Metamask.
- *
- * AxelarChainIds must be specified in SourceChain{} and are used in ./source-chain-configs.ts::SourceChainConfigs{} as <asset>::<network>::id values.
- *
- *  Values not included as keys are assumed to be the same across chainlist and Axelar.
+
+/** Maps Axelar chain id agruments => source chain ids. 
+ *  SourceChain (IDs) are used in ./source-chain-configs.ts::SourceChainConfigs{} as <asset>::<network>::id values.
+ *  Axelar Chain IDs are accepted as arguments in Axelar's APIs.
+ *  Mainnet Docs: https://docs.axelar.dev/dev/build/chain-names/mainnet
+ *  Testnet Docs: https://docs.axelar.dev/dev/build/chain-names/testnet
+ *  Testnet API: https://axelartest-lcd.quickapi.com/axelar/nexus/v1beta1/chains?status=1
  */
-export const EthClientChainIds_AxelarChainIdsMap: {
-  [ethClientChainIds: string]: SourceChain;
+export const AxelarChainIds_SourceChainMap: {
+  [axelarChainIds: string]: SourceChain;
 } = IS_TESTNET
   ? {
-      "Aurora Testnet": "aurora",
-      "Avalanche Fuji Testnet": "Avalanche",
-      "Binance Smart Chain Testnet": "binance",
-      "Goerli Test Network": "ethereum-2",
-      "Fantom Testnet": "Fantom",
-      "Moonbase Alpha": "Moonbeam",
-      Mumbai: "Polygon",
+      "aurora": "Aurora Testnet",
+      "Avalanche": "Avalanche Fuji Testnet",
+      "binance": "BSC Testnet",
+      "ethereum-2": "Goerli Testnet",
+      "Fantom": "Fantom Testnet",
+      "Moonbeam": "Moonbase Alpha",
+      "Polygon": "Mumbai"
     }
   : {
-      "Avalanche C-Chain": "Avalanche",
-      "Binance Smart Chain Mainnet": "binance",
-      "Ethereum Main Network": "Ethereum",
-      "Fantom Opera": "Fantom",
-      "Moonbeam Mainnet": "Moonbeam",
-      "Polygon Mainnet": "Polygon",
+      "Avalanche": "Avalanche",
+      "binance": "Binance Smart Chain",
+      "Ethereum": "Ethereum",
+      "Fantom": "Fantom",
+      "Moonbeam": "Moonbeam",
+      "Polygon": "Polygon"
     };
 
+
+/** Maps eth client chainIDs => source chain ids.
+ *
+ *  ethClientChainIDs must be specified in ../ethereuem/types.ts::ChainNames{} to map the name to a chainID, which is in turn used to add the network to EVM-compatible wallets, like Metamask.
+ *  SourceChain (IDs) are used in ./source-chain-configs.ts::SourceChainConfigs{} as <asset>::<network>::id values.
+ */
+export const EthClientChainIds_SourceChainMap: {
```

Comment:
Let's keep the location of these config definitions organized. Since SourceChain is no longer specific to Axelar (it's more of a global definition now, I'd say), let's move `EthClientChainIds_SourceChainMap` and `SourceChain` it to bridge-info.ts

Reply:
Done